docs/changelog.md

```diff
@@ -20,9 +23,9 @@ permalink: /changelog/
 * **feature-top-sites**
   * ⚠️ **This is a breaking change**: The existing data class `TopSite` has been converted into a sealed class. [#11483](https://github.com/mozilla-mobile/android-components/issues/11483)
   * Extend `DefaultTopSitesStorage` to accept a `TopSitesProvider` for fetching top sites. [#11483](https://github.com/mozilla-mobile/android-components/issues/11483)
-  
+
```

Comment:
I'm not sure what happened here, it appears after a rebase, but if i'm trying to revert this change locally and commit it says that this file is identical with the main branch version